CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # Braintree iOS SDK Release Notes
 
+## unreleased
+* BraintreePayPalNativeCheckout (BETA)
+  * Fix CocoaPods bug emitting "Cannot find interface declaration" error
```

Comment:
I think it would be helpful for future us to describe the scenario that caused this error (the nested subclassing from ObjC: ObjC: Swift). We could add a note in the CHANGELOG or even in this PR description too just so we have a trace of it if a similar issue happens to us in the future.